src/components/ui/Footer/Footer.tsx

```diff
@@ -0,0 +1,377 @@
+          fill="#0923BA"
+        />
+      </svg>
+      {/* Oblong Shape*/}
+
+      <div
+        id="main"
+        className={`${footerSection({ background: FOOTER_BACKGROUND_ENUM.BLUE })} ${footerLink()} w-full flex flex-col px-[50px] md:px-[50px] items-center`}
+      >
+        <div className="w-full flex items-center">
+          <div
+            id="Logo-Text"
+            className="flex flex-col xl:flex-row items-center xl:items-center xl:justify-start justify-center w-full"
+          >
+            {/* Logos */}
+
+            <div id="logo " className="flex flex-row items-center">
+              <Image
+                src={UniversitySealLogo}
+                width={122}
+                height={122}
+                className="mt-10 mr-[20px]"
+                alt="University Seal Logo"
+              />
+              <Image
+                src={SamahanLogoWhite}
+                width={122}
+                height={122}
+                className="mt-10"
+                alt="Samahan Logo White"
+              />
+            </div>
+            <div
+              id="Text"
+              className="mt-5 xl:mt-10 lg:ml-4 text-center xl:text-left"
+            >
+              <h3 className="text-2xl font-semibold font-inter">
+                Samahan ng mga Mag-aaral ng Pamantasan
+              </h3>
+
+              <h3 className="text-2xl font-semibold font-inter mb-2">
+                ng Ateneo de Davao
+              </h3>
+              <p className="font-inter" style={{ fontSize: '14px' }}>
+                The Student Government of College Unit of the Ateneo de Davao
+                University
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div
+          id="content"
+          className="font-inter flex flex-col lg:flex-row mt-[42px] w-full ml-start mr-auto items-center lg:items-start lg:justify-between"
+        >
+          <div
+            id="left"
+            className="flex flex-col lg:flex-row lg:space-x-[3.125rem]"
+          >
+            {/* Quick Links */}
+            <div
+              id="links"
+              className="flex flex-col space-y-3 text-center lg:text-left mb-[3.125rem] lg:mb-0"
+            >
+              <h3 className="font-bold" style={{ fontSize: '16px' }}>
+                Quick Links
+              </h3>
+              {/* Map Function */}
+              <ul className="space-y-[0.938rem]">
+                {quickLinks.map((link, index) => (
+                  <li key={index}>
+                    <Link href={link.href} legacyBehavior>
+                      <a
+                        className={footerLink({
+                          size: LINK_SIZE_ENUM.REGULAR,
+                          weight: LINK_WEIGHT_ENUM.SUPER_THIN,
+                        })}
+                      >
+                        {link.label}
+                        {link.icon && link.icon}
+                      </a>
+                    </Link>
```

Comment:
Remove <a> tag. Redundant na kasi may Link component na. Do this for others as well pls. Thanks!